rest-server/engines/src/main/scala/com/actionml/engines/navhinting/NavHintingDataset.scala

```diff
@@ -213,6 +214,7 @@ case class NHNavEventProperties(
 
 case class NHNavEvent(
     event: String,
+    entityType: String,
     entityId: String,
```

Comment:
is this required?

Reply:
I suppose https://github.com/actionml/harness/blob/develop/java-sdk/example/data/nh-pferrel-events-urls.json